src/streaming.ts

```diff
@@ -17,6 +18,7 @@ type FetchFunction = (
 export class StreamProcessor {
   static readonly CONNECTED = 1;
   static readonly RETRYING = 2;
+  static readonly CLOSED = 3;
```

Comment:
Don't change in this PR, but this really aught to be an enum, not readonly static vars